PeerioTechnologies/peerio-mobile
# fix: mobile admin state screen (content scrollable, chat opens multiple times)

#### Relevant info and issue/PR links 
 
#### Testing instructions  
scroll is enabled if content is clipped
if tapping multiple times on help button, it should open one chat

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
